pyproject.toml

```diff
@@ -354,7 +354,6 @@ module = [
     "sentry.middleware.auth",
     "sentry.middleware.ratelimit",
     "sentry.middleware.superuser",
-    "sentry.models.auditlogentry",
```

Comment:
I'm confused, it looks like a ton of types are missing in this module, why doesn't mypy complain more?

Reply:
the top list is things that are _failing_ the type checker -- the bottom list requires all things to be typed